common/app/layout/Sublink.scala

```diff
@@ -60,8 +61,9 @@ case class Sublink(
   url: EditionalisedLink,
   cardStyle: CardStyle,
   mediaType: Option[MediaType],
-  pillarName: String,
-  contentType: String
+  pillar: Option[Pillar],
+  contentType: String,
```

Comment:
would it make sense to store the `DotcomContentType` instance here rather than a `String`?